R/courses.R

```diff
@@ -1366,7 +1367,9 @@ c_41 <- function(input_df=usheUtils::fake_course_df) {
 
   output_df <- input_df %>%
     # Calculate intermediate fields
-    mutate( c_title = course_title) %>%
+    mutate( c_title_intermediate_1 = str_replace_all(course_title, "([,.;:-])", "") ) %>%
+    mutate( c_title_intermediate_2 = str_replace_all(c_title_intermediate_1, "&", "and") ) %>%
+    mutate( c_title = c_title_intermediate_2) %>%
```

Comment:
I think these operations would be good to perform over every data column that we submit to USHE. Not sure how that would look. But I just want to make note of that here.